lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -403,6 +408,12 @@ enum SemanticRoleKind {
   /// Denotes a header.
   header,
 
+  /// An individual tab button.
+  tab,
+
+  /// A main content for a tab.
+  tabPanel,
```

Comment:
Curious why `tabBar` is missing. Does `tablist` ARIA role not work for it?

Reply:
oh thanks for pointing out, I thought there isn't an equivalence